### ISSUE TITLE
The library should allow to sort data in custom order

### ISSUE THREAD

**Issue by jaepetto** [webdacjs/simple-pivot-table#21]
As a developer, I may want to display data in a given order. Typically, entries in résumé are displayed from the latest to the oldest:
| Year | Activity | Note |
| ---- | -------- | ----- |
| 2021 | A | ... |
| 2021 | B | ... |
| 2020 | A | ... |
| 2020 | C | ... |

For the time being, the library automatically sorts the data.
It would be nice if it was:
1. keep the data in the order provided
2. provide some property to allow sorting operations like in lodash: `_.orderBy(data, ['Year', 'Activity'], ['desc', 'asc'])`

**Comment by webdacjs** [comment]
Thank you for the feature request @jaepetto that def. sounds like a good idea and I will implement it. I do indeed sort the data when doing the grouping, but I see the point to order the resulting data based on the parameters required by the developer. I will assign it to me.

**Comment by webdacjs** [comment]
This was implemented and released in v2.7.0 (already published to NPM) @jaepetto. If you are curious please look at https://github.com/webdacjs/simple-pivot-table/pull/22. Based on your previous comment the way it works is you can pass an additional parameter `orderBy`, containing an array of objects with the field and the direction.

So in your case it would be:

```
....
  orderBy={[
      {field: 'Year', order: 'desc'},
      {field: 'Activity', order: 'asc'}
    ]
  }
...

```
Thanks again for the request and I'm closing this issue for now. Feel free to reopen it if something doesn't work as expected.
